moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -54,7 +54,7 @@ namespace core
  * valid paths from paths with large joint space jumps. */
 static const std::size_t MIN_STEPS_FOR_JUMP_THRESH = 10;
 
-const std::string LOGNAME = "robot_state";
+static const rclcpp::Logger LOGNAME = rclcpp::get_logger("moveit.robot_state");
```

Comment:
`LOGNAME` no longer makes sense here. Please change variable name

Reply:
change to what? william said the logger handler is basically just a string, so i think it does make sense. this doesn't connect to a node.

Reply:
> LOGNAME no longer makes sense here.

I had the same initial impulse and then reasoned like Dave to keep it.
However, a better name would be `LOGGER`.

Reply:
updated

Reply:
I would just put a word of caution here, using all caps for a variable name can be dangerous because those are usually reserved for macros. If you were following our style it would be `g_logger`, or the Google style might indicate `kLogger`. `LOGGER` might have made more sense when it was actually a `#define` before, but now it's a global variable.